packages/components/src/components/page-error/page-error.tsx

```diff
@@ -34,6 +36,7 @@ const PageError = ({
     redirect_urls,
     should_clear_error_on_click,
     setError,
+    should_redirect,
```

Comment:
can we set `should_redirect` to true to make sure it doesn't break anywhere else?

Reply:
@balakrishna-deriv we do it in the props of `<ErrorComponent` which is the parent of `<PageError` component. I think it would be enough, wouldn't it?

Reply:
<img width="355" alt="image" src="https://user-images.githubusercontent.com/56330681/228147404-ccf093a7-89f2-408b-a0cc-a1703c4a1d77.png">
PageError is being used in multiple places and not just ErrorComponent. To ensure we don't break anything, we can consider setting the default prop to true.

Reply:
done✅